src/test/java/org/numenta/nupic/RunLayer.java

```diff
@@ -105,17 +110,51 @@ public Layer(Connections c, MultiEncoder encoder, SpatialPooler sp,
             this.tm = tm;
             this.classifier = cl;
             
-            try {
-                this.encFile = new PrintWriter(new FileWriter(encFilePath));
-            }catch(Exception e) {
-                e.printStackTrace();
-            }
+//            try {
+//                this.encFile = new PrintWriter(new FileWriter(encFilePath));
+//            }catch(Exception e) {
+//                e.printStackTrace();
+//            }
```

Comment:
I kind of want to preserve all of these... I have gone back and forth so much during all of this debugging that it just has become easier to leave this stuff in.

If you really thing it shouldn't be in here, I'm fine with taking them out because I just may be past all of this kind of debugging actually...